vespalib/src/vespa/vespalib/stllike/vector_map.h

```diff
@@ -82,6 +79,19 @@ void swap(vector_map<K, V, LT> & a, vector_map<K, V, LT> & b)
     a.swap(b);
 }
 
+template< typename K, typename V, typename LT >
+V &
+vector_map<K, V, LT>::operator[](const K &key) {
+    LT lt;
+    iterator f = std::lower_bound(begin(), end(), key, KeyOrder());
+    if (f == end()) {
+        _ht.template emplace_back(key, V());
```

Comment:
how much slower is emplace at end() compared to emplace_back?

Reply:
emplace_back, and push_back has inlined fast path.
emplace, and insert does not.